File: submissions/sm_001_aalind/week_03/day_3/session_2/count_vowels.js

function count_vowels(word) {
  var count = 0;

  for(var i = 0;i < word.length;i++) {
    if(word[i] == "a" || word[i] == "e" || word[i] == "i" || word[i] == "o" || word[i] == "u") {
      count += 1;
    }
  }

  return count;
}

console.log(count_vowels("aalind sharma"))

File: submissions/sm_010_dipanshu/week_13/day_4/session_1/count_vowels.py
vowels = {
    "a": True,
    "A": True,
    "e": True,
    "E": True,
    "i": True,
    "I": True,
    "o": True,
    "O": True,
    "u": True,
    "U": True,
}

words = ["dipanshu", "DIPANSHU", "DiPaNsHu", "gaurav", "GAURAV", "gAuRaV", "aalind", "AALIND", "aAlInD"]
result = []

for word in words:
    count = 0
    for char in word:
        if char in vowels:
            count += 1
    result.append(count)

print(result)
